tools/util.wdl

```diff
@@ -460,11 +465,12 @@ for read in bam:
 
     cur_quals = read.query_alignment_qualities  # array of phred scores
     for qual in cur_quals:
-        tot_quals[qual] += 1
-        if read.is_unmapped:
-            unmapped_quals[qual] += 1
-        else:
-            mapped_quals[qual] += 1
+        if not only_mid:
```

Comment:
It may just be the GitHub browser, but assuming this is the entire `for` loop, shouldn't the `if` go outside?

Reply:
That is:
```
if not only_mid:
    for qual in cur_quals:
```

Reply:
You are correct! Logically equivalent but **not** equivalent for efficiency. Fixing now.